promtimer/promtimer.py

```diff
@@ -309,15 +326,15 @@ def main():
     prometheus_base_port = grafana_port + 1
     live_cluster = args.cluster or args.nodes
 
-    if not live_cluster:
-        stats_sources = cbstats.CBCollect.get_stats_sources(prometheus_base_port)
-        if not stats_sources:
-            sys.exit(1)
-        times = cbstats.CBCollect.compute_min_and_max_times(stats_sources)
-        min_time = datetime.datetime.fromtimestamp(times[0]).isoformat()
-        max_time = datetime.datetime.fromtimestamp(times[1]).isoformat()
-        refresh = ''
-    else:
+    if live_cluster and args.backup_archive_path:
+        print("--cluster and --nodes are mutually exclusive ...")
+        sys.exit(1)
+
+    backup_archive_mode = False
```

Comment:
This is not needed as `backup_archive_mode` is always equal to `args.backup_archive_path` when compared as a boolean. Please remove.

Reply:
That's a good point. Not sure how this slipped by me 😞